examples/file-transfer/src/__tests__/acceptance/test-helper.ts

```diff
@@ -42,6 +42,7 @@ export interface AppWithClient {
 }
 
 export function getSandbox() {
-  const sandbox = new TestSandbox(path.resolve(__dirname, '../../../.sandbox'));
+  // dist/.sandbox
```

Comment:
Can you please improve the comment and explain why it's ok to use the same directory `dist/.sandbox` when running the tests in parallel?

Reply:
Sorry for the confusion. We do use a unique subdir now. I fixed the comment.